pkg/evs/controllerserver.go

```diff
@@ -191,32 +191,66 @@ func (cs *ControllerServer) ControllerGetVolume(_ context.Context, req *csi.Cont
 	return &response, nil
 }
 
+type VolumeAttachmentStatus int32
+
+const (
+	VolumeAttachmentAvailable VolumeAttachmentStatus = iota + 1
+	VolumeAttachmentAttachingMatchServer
+	VolumeAttachmentAttachingNotMatchServer
+	VolumeAttachmentInUseMatchServer
+	VolumeAttachmentInUseNotMatchServer
+	VolumeAttachmentError
```

Comment:
how about the following definition:
`
VolumeNotAttached VolumeAttachmentStatus = iota + 1
VolumeAttachingCurrentServer
VolumeAttachingOtherServer
VolumeAttachedCurrentServer
VolumeAttachedOthreServer
VolumeAttachError
`

Reply:
seems clearer！